service/controller/v25/cloudformation/cloud_formation.go

```diff
@@ -29,11 +29,11 @@ func New(config Config) (*CloudFormation, error) {
 // DescribeOutputsAndStatus returns stack outputs, stack status and error. The
 // stack status is returned when the error is nil or the error is matched by
 // IsOutputsNotAccessible.
-func (c *CloudFormation) DescribeOutputsAndStatus(stackName string) ([]*cloudformation.Output, string, error) {
+func (c *CloudFormation) DescribeOutputsAndStatus(stackName string) ([]Output, string, error) {
```

Comment:
We abstract certain external functionality but were still relying on the external types. This caused an asymmetry in the dependency graph and caused import weirdness where the abstraction got used. The inspiration here came from the ebs service implementation.